src/rules/no-extraneous-dependencies.js

```diff
@@ -268,6 +291,7 @@ module.exports = {
           'packageDir': { 'type': ['string', 'array'] },
           'includeInternal': { 'type': ['boolean'] },
           'includeTypes': { 'type': ['boolean'] },
+          'considerInParents': { 'type': ['array'] },
```

Comment:
```suggestion
          considerInParents: {
             type: 'array',
             uniqueItems: true,
             additionalItems: false,
             items: {
               type: 'string',
               enum: [
                'prod',
                'dev',
                'peer',
                'bundled',
                'optional',
              ],
```

Reply:
- You removed the quotes around considerInParents were removed, but all the others in the schema have quotes - is there a reason for the difference or should it be one or the other consistently?
- It might be a good idea to update the schema for the other items to specify that they're arrays of strings? Though that would presumably be best in another PR?

Reply:
doesn't matter here, but i'll be enabling an eslint rule to properly omit the quotes where possible :-)

While it would be great to do that, that'd be a breaking change, and it's not worth doing one just for that.